src/build/nextStepCalculatorGenerator.ts

```diff
@@ -1,27 +1,31 @@
-import {Locale, IEdges} from '../types/index'
+// TODO: handle steps that don't have translations
+import {Locale} from '../types/index'
 import {INext} from '../types/chapter'
+import {TMadrassahPageQueryQuery} from '../graphqlTypes'
 
 export interface INextStepTrack {
   id: string
   slug: string
-  courseSet: IEdges<{
+  courses: {
```

Comment:
You can break courses into type, and every nested object in this Interface, we may use these types somewhere else